src/components/pdfPage/PdfPage.tsx

```diff
@@ -119,6 +122,11 @@ const PdfPage: FC<IPdfPageProps> = (props) => {
   const isBboxSelected = (bbox: IBbox) => props.activeBboxIndex === bbox.index;
   const isRelated = (bbox: IBbox) => props.groupId ? bbox.groupId === props.groupId && !isBboxSelected(bbox) : false;
 
+  useEffect(() => {
+    props.onWarning?.(
```

Comment:
Let's call the props.onWarning only if the message is not null (null message mean there is no warning)